src/repo.rs

```diff
@@ -1,41 +1,52 @@
 use crate::common::DHTEntity;
 use async_stream::stream;
-use eyre::Result;
+use anyhow::{Result, anyhow};
 use futures_core::stream::Stream;
 use std::sync::Arc;
+use std::io::ErrorKind;
+use tokio::sync::{mpsc, broadcast};
+use bytes::{Bytes, BytesMut, BufMut};
 use veilid_core::{
-    CryptoKey, CryptoSystemVLD0, CryptoTyped, DHTRecordDescriptor, RoutingContext, SharedSecret,
+    CryptoKey, CryptoSystemVLD0, CryptoTyped, DHTRecordDescriptor, RoutingContext, SharedSecret, VeilidAPI, Target, VeilidUpdate,
 };
+use iroh_blobs::Hash;
+use veilid_iroh_blobs::iroh::VeilidIrohBlobs;
+
+use crate::constants::{ASK, DATA, DONE, YES, NO, ERR};
```

Comment:
These should probably be hidden in the impl details in the veilid-iroh-blobs module.